src/main/resources/application.conf

```diff
@@ -28,6 +28,22 @@ datastax-java-driver {
     }
   }
   basic.request.timeout = 10 seconds
+  advanced.request-tracker {
```

Comment:
nit: should come before `basic.request.timeout` to keep things in alphabetic order.

Reply:
Changed in [this commit](https://github.com/stargate/jsonapi/pull/754/commits/b70a3266bab05de5f9c68f73bd0963e3a3e399f2).